changelog/1.24.0.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "1.24.0"
+description: "Released on 10/21/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.24.0.
+
+### Features ✨
+
+- web: added ability to pull images from private Amazon ECR repositories.
+- web: added alert to notify users when workspace disks are full.
+- web: added ability for end-users to set auto-off behavior for their
+  workspaces.
+- web: added information regarding applications used to the audit log.
+- web: updated the in-product changelog to display information for multiple
+  versions of Coder.
+- web: added ability to set the background color for all in-product banners with
+  a color picker.
+- infra: added auto-injection of certificates needed by `coderd`.
```

Comment:
@Emyrk is there a better way this could be phrased? Maybe `added auto-injection of `coderd`'s certificate into the workspace coder agent certificate trust store.` or something

Reply:
It only affects `coder agent` when connecting back to the access-url.

> Auto-inject tls certificates into workspaces for secure trusted communication with coderd 

I'm not exactly sure on the phrasing :thinking: